common/tools/dev-tool/src/commands/samples/checkNodeVersions.ts

```diff
@@ -202,6 +202,13 @@ export const commandInfo = makeCommandInfo(
       description: "A comma separated list of node versions to use",
       default: "8,10,12"
     },
+    "node-version": {
```

Comment:
I'm not too opposed to this, but think it will be simpler to just pick one or the other of node-versions and node-version. You could even have a single option that is the best of both.

`dev-tool samples check-node-versions --node-version "10,12" --node-version "14"`.

Just by treating each entry as comma separating and concatenating all the way through the multiple specifications of the option.

Reply:
Yeah I just wanted to make sure the the argument name is consistent with its behavior (whether it is plural or not). I think the other benefit of having it like this is that this change will not be breaking/annoying for people already used to the plural name.